src/hct_mis_api/apps/registration_data/fixtures.py

```diff
@@ -64,3 +66,49 @@ class Meta:
         before_now=True,
         tzinfo=utc,
     )
+
+
+def generate_rdi() -> None:
+    ba = BusinessArea.objects.get(slug="afghanistan")
```

Comment:
fixture!

Reply:
this one as well used only in `initdemo.py`